App.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React from "react";
+import styled from "styled-components/native";
+import Quotes from "./components/Quotes";
+import { Text } from "react-native";
```

Comment:
since you're not using this you don't need to import it.  when setting up the styled components it'll sort that out itself to "find" the right element.